### ISSUE TITLE
Add GLTFEnabled to SimulatorFeatures

### ISSUE THREAD

**Issue by RunitaiLinden** [secondlife/viewer#1129]
This will act as the feature flag for enabling GLTF Scene drag/drop (#1128)

When connected to GLTF enabled regions (i.e. PMFP channel on aditi), Develop->GLTF->Open should be enabled.
Otherwise it should be disabled.

**Comment by RunitaiLinden** [comment]
Viewer side integrated to GLTF Dev

Server side waiting on https://github.com/secondlife/server/pull/1020

**Comment by AtlasLinden** [comment]
Regions on Aditi using PMFP channels include Rumpus Room 1-4 and 2048-2051

**Comment by DmitrySProductengine** [comment]
Deferred QA.
Verified on the Second Life Release 7.1.9.9865199239 (64bit) on Win10/OSX in the scope of [IQA #282](https://github.com/secondlife/iqa/issues/282).

Tested on Aditi:
1. PMFP - Rumpus Room 2048 (PMFP 2024-06-25.9663125755).
2. SLS - Product Engine Trunk (Second Life Server 2024-06-11.9458617693).
3. WebRTC - webRTC1 (WebRTC-Voice 2024-06-24.9652733849).

In all regions (not just PMFP), the new feature was enabled (Open was available).
Theoretically, it is possible that PMFP (GLTF-enabled regions) has already been added to SLS regions and this behavior is expected.
This ticket does not indicate the exact criteria (or examples) of regions where the new feature should be disabled.

But there is also a difference in the behavior of this new feature on different simulators:
1. This feature is active on the SLS simulator, allowing you to open GLTF, but the “Save As” and “Upload” functions are inactive.
2. On the PMFP simulator, all 3 functions are active - “Open”, “Save as” and “Upload”.

Due to the described difference in behavior and due to the difficulty of determining the regions where the feature should be disabled, this bug was moved to the Needs info.
Perhaps this behavior is erroneous and the ticket needs to be reopened.

See attached screenshots:
1. 7.1.9.9865199239 - Aditi - PMFP - all 3 functions enabled
<img width="1165" alt="7 1 9 9865199239 - Aditi - PMFP - all 3 functions enabled" src="https://github.com/user-attachments/assets/76902574-fc30-445c-8403-e82da46be29f">

2. 7.1.9.9865199239 - Aditi - SLS - only Open enabled
<img width="1171" alt="7 1 9 9865199239 - Aditi - SLS - only Open enabled" src="https://github.com/user-attachments/assets/6046e6bb-18c5-4d45-87d5-30597f0c8477">

3. 7.1.9.9865199239 - Aditi - WebRTC - only Open enabled
<img width="1172" alt="7 1 9 9865199239 - Aditi - WebRTC - only Open enabled" src="https://github.com/user-attachments/assets/62103592-0331-41fa-a8f2-a2893f933596">

A new crash bug was also issued when trying to open a test GLTF with 2k textures - https://github.com/secondlife/viewer/issues/2041.

**Comment by AtlasLinden** [comment]
Setting this to Pending Release as GLTF previews are only available client side.
Save As / Upload are covered here https://github.com/secondlife/viewer/issues/1638 and will verified again when a place outside the Develop menu is defined for these options.
